integration-tests/test_common_specs.py

```diff
@@ -1,31 +1,54 @@
 import json
 import os
 import pytest
+import platform
 
 pytestmark = pytest.mark.usefixtures("register_subman")
 
 
 def test_common_specs(insights_client, tmp_path):
-    """Verify that the specified specs can be collected and parsed as expected."""
-    common_specs = [
-        "insights.specs.Specs.date.json",
-        "insights.specs.Specs.dmidecode.json",
-        "insights.specs.Specs.fstab.json",
-        "insights.specs.Specs.hostname.json",
-        "insights.specs.Specs.hosts.json",
-        "insights.specs.Specs.installed_rpms.json",
-        "insights.specs.Specs.ip_addresses.json",
-        "insights.specs.Specs.ls_dev.json",
-        "insights.specs.Specs.lscpu.json",
-        "insights.specs.Specs.lspci.json",
-        "insights.specs.Specs.meminfo.json",
-        "insights.specs.Specs.mount.json",
-        "insights.specs.Specs.mountinfo.json",
-        "insights.specs.Specs.ps_auxcww.json",
-        "insights.specs.Specs.redhat_release.json",
-        "insights.specs.Specs.uname.json",
-        "insights.specs.Specs.yum_repos_d.json",
-    ]
+    """
+    Verify that the specified specs can be collected and parsed as expected.
+    Ref: https://issues.redhat.com/browse/RHINENG-10737
+    """
+    if platform.processor() == "x86_64":
+        common_specs = [
+            "insights.specs.Specs.date.json",
+            "insights.specs.Specs.dmidecode.json",
+            "insights.specs.Specs.fstab.json",
+            "insights.specs.Specs.hostname.json",
+            "insights.specs.Specs.hosts.json",
+            "insights.specs.Specs.installed_rpms.json",
+            "insights.specs.Specs.ip_addresses.json",
+            "insights.specs.Specs.ls_dev.json",
+            "insights.specs.Specs.lscpu.json",
+            "insights.specs.Specs.lspci.json",
+            "insights.specs.Specs.meminfo.json",
+            "insights.specs.Specs.mount.json",
+            "insights.specs.Specs.mountinfo.json",
+            "insights.specs.Specs.ps_auxcww.json",
+            "insights.specs.Specs.redhat_release.json",
+            "insights.specs.Specs.uname.json",
+            "insights.specs.Specs.yum_repos_d.json",
+        ]
+    else:
+        common_specs = [
+            "insights.specs.Specs.date.json",
+            "insights.specs.Specs.fstab.json",
+            "insights.specs.Specs.hostname.json",
+            "insights.specs.Specs.hosts.json",
+            "insights.specs.Specs.installed_rpms.json",
+            "insights.specs.Specs.ip_addresses.json",
+            "insights.specs.Specs.ls_dev.json",
+            "insights.specs.Specs.lscpu.json",
+            "insights.specs.Specs.meminfo.json",
+            "insights.specs.Specs.mount.json",
+            "insights.specs.Specs.mountinfo.json",
+            "insights.specs.Specs.ps_auxcww.json",
+            "insights.specs.Specs.redhat_release.json",
+            "insights.specs.Specs.uname.json",
+            "insights.specs.Specs.yum_repos_d.json",
+        ]
```

Comment:
Looks good to me. Just a tiny suggestion, how about making the following change to make the difference between `x86_64` and other archs a bit clear?
```python
     common_specs = [
            "insights.specs.Specs.date.json",
            "insights.specs.Specs.fstab.json",
            "insights.specs.Specs.hostname.json",
            "insights.specs.Specs.hosts.json",
            "insights.specs.Specs.installed_rpms.json",
            "insights.specs.Specs.ip_addresses.json",
            "insights.specs.Specs.ls_dev.json",
            "insights.specs.Specs.lscpu.json",
            "insights.specs.Specs.meminfo.json",
            "insights.specs.Specs.mount.json",
            "insights.specs.Specs.mountinfo.json",
            "insights.specs.Specs.ps_auxcww.json",
            "insights.specs.Specs.redhat_release.json",
            "insights.specs.Specs.uname.json",
            "insights.specs.Specs.yum_repos_d.json"
    ]

    if platform.processor() == "x86_64":
        common_specs.extends([
            "insights.specs.Specs.dmidecode.json",
            "insights.specs.Specs.lspci.json",
        ])
```

Reply:
Thanks. common_specs.extends => common_specs.extend